Lib/test/support/import_helper.py

```diff
@@ -169,9 +193,14 @@ class CleanImport(object):
 
         with CleanImport("foo"):
             importlib.import_module("foo") # new reference
+
+    If "usefrozen" is False (the default), any stdlib source module will
+    always be imported from its .py file, even if the module has been
+    frozen.  The only exception is essential modules (like
```

Comment:
Can you rewrite this without referring to .py files? I keep thinking "or .pyc file" when I read this. Maybe just state that it disables the frozen module importer except for essential modules?